docs/start/configure-qiskit-local.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Configure Qiskit locally
+description: Configure Qiskit on your local machine
+
+---
+# Configure Qiskit locally
+
+After Qiskit is installed and running, there are some optional steps you can take to change the default Qiskit behavior.
+
+## User configuration file
+
+The main location for local configuration of Qiskit is the user configuration file. This is an .ini-format file that can be used to change Qiskit default settings.
+
+Example:
+
+ ```text
+[default]
+circuit_drawer = mpl
+circuit_mpl_style = default
+circuit_mpl_style_path = ~:~/.qiskit
+state_drawer = hinton
+transpile_optimization_level = 3
+parallel = False
+num_processes = 15
+ ```
+
+By default this file is in `~/.qiskit/settings.conf` but the path can be overridden with the QISKIT_SETTINGS environment variable.
+
+## Available options
+
+- `circuit_drawer`: Changes the default backend for the circuit drawer. It can be set to `latex`, `mpl`, `text`, or `latex_source`. When the output kwarg is not explicitly set, this drawer backend is used.
+- `circuit_mpl_style`: The default style sheet used for the mpl output backend for the circuit drawer. Valid values are `default` or `bw`.
+- `circuit_mpl_style_path`: The paths to have the circuit drawer use to look for JSON style sheets when using the mpl output mode.
+- `state_drawer`: This is used to change the default backend for the state visualization draw methods. Valid values are `repr`, `text`, `latex`, `latex_source`, `qsphere`, `hinton`, or `bloch`. When the output kwarg is not explicitly set on the :meth:`~qiskit.quantum_info.DensityMatrix.draw` method, the specified output method is used.
```

Comment:
Are we supposed to be able to see the :meth: tags  and the prepended ~ throughout? Or is this leftover formatting from the migration?

Reply:
I guess?  I thought it looked weird, but I checked and you can see them in the original link I was sent too: https://github.com/Qiskit/qiskit/blob/main/docs/configuration.rst#id1

Reply:
Hm. Looks like shards of .rst to me... I found [this page](https://stackoverflow.com/questions/34848253/whats-the-difference-in-behaviour-between-func-and-meth-roles-in-python-sph) that seems to suggest that the html output shouldn't show the tags (also for :func:)??

Reply:
Oh yeah those :meth: things definitely shouldnt be there, it must have been an error in the originaly doc, instead it should be a link to that method in the api ref

Reply:
I tried to add links for all of them; hopefully they're going to the right spot.